Assets/Scripts/DesignObjects/Floater.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+/** Floater, public class
+ * This class is used for Designing elements in the game.
+ * When a GameObject is using this class, we allows him to "float" up to down and down to up. 
```

Comment:
And UP and DOWN and UP and DOWN !!

Go to the party ROCKSSS.

Reply:
Mdr